app-client/src/main/java/com/keycloak/cdc/service/AppService.java

```diff
@@ -0,0 +1,47 @@
+package com.keycloak.cdc.service;
+
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import org.springframework.web.servlet.mvc.method.annotation.SseEmitter;
+
+@Service
+@EnableScheduling
+public class AppService {
```

Comment:
From the comment above on the `\notification` endpoint, this class then serves no function

Reply:
Resolved